### DISCUSSION TITLE
Add native support for acronyms

### DISCUSSION THREAD

**Discussion by Minh-AnhHuynh** [quarto-dev/quarto-cli#2425] [Feature Requests]
Hello,

I was writing my manuscript and noticed there is a lack of implementation for an automatic, ordered list of acronyms. In Latex, you have package glossary, and in R markdown, package acronymsdown. It would be nice to have a similar support native in quarto so that producing manuscripts would be easier.

Cheers,
Minh-Anh

**Comment by lcnbr** [comment]
see #1763
